packages/mobile/src/invite/saga.ts

```diff
@@ -86,7 +86,9 @@ export function* sendInvite(
       ? 'sendFlow7:inviteWithEscrowedPayment'
       : 'sendFlow7:inviteWithoutPayment'
     const message = i18n.t(messageProp, {
-      amount: amount?.toString(),
+      amount: amount?.toFixed(2),
```

Comment:
Why the optional chaining on `amount`? I notice it was there before but it looks like a required param so maybe we should remove it